profiles/bicycle.lua

```diff
@@ -402,7 +402,7 @@ end
 
 function turn_function (angle)
   -- compute turn penalty as angle^2, with a left/right bias
-  k = turn_penalty/(90.0*90.0)
```

Comment:
given that this is technically a bug in the API, we should add a comment here stating where the factor 10 comes from.